array-map/main.js

```diff
@@ -0,0 +1,18 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language',
+];
+
+const doubled = numbers.map(numbers => numbers * 2);
+console.log(doubled);
+const prices = numbers.map(numbers => `$${numbers}.00`);
+console.log(prices);
+const upperCased = languages.map(languages => languages.toUpperCase());
+console.log(upperCased);
+const firstLetters = languages.map(languages => languages.charAt(0));
```

Comment:
These arrow functions all use a plural name for their parameter, but they are only being passed a single thing, so a singular name would be better.